pages/common/surge.md

```diff
@@ -0,0 +1,19 @@
+# Surge
+
+> Simple, single-command web publishing.
+
+- Upload a new site:
```

Comment:
Where does it upload to?

Reply:
It uploads to surge.sh servers - clarified that in the most recent change